CMakeLists.txt

```
@@ -37,10 +37,20 @@ macro(set_var_to_condition var)
     endif()
 endmacro()
 
+get_property(_GENERATOR_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
```

Normally we use the "MIOPEN" prefix in such cases, isn't it?
```suggestion
get_property(MIOPEN_GENERATOR_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
```

---

@apwojcik if you agree we can make this change quickly :)